package/bin/user_aggregates_helper.py

```diff
@@ -63,9 +81,11 @@ def stream_events(inputs: smi.InputDefinition, event_writer: smi.EventWriter):
             checkpointer_key_name = input_name.split("/")[-1]
             now = datetime.now()
             # No checkpoint? Default it to four years ago per API docs
+            # AN 2025-10-14: Changed default start date to 5 minutes ago to reduce data volume on first run
             last_checkpoint = (
                 kvstore_checkpointer.get(checkpointer_key_name)
-                or (now - relativedelta(years=4)).strftime("%Y-%m-%dT%H:%M:%SZ")
+                or (datetime.now() - relativedelta(minutes=5)).timestamp()
+                #(now - relativedelta(years=4)).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Cleaned up comment and used `timedelta` instead of `relativetime` to calculate proper time and keep code consistency
```suggestion
            # No checkpoint? Default to start date of 5 minutes ago to reduce data volume on first run
            last_checkpoint = (
                kvstore_checkpointer.get(checkpointer_key_name)
                or (now - timedelta(minutes=5)).timestamp()
```